src/main/plugins/org.dita.base/build_preprocess_template.xml

```diff
@@ -20,6 +20,7 @@ See the accompanying LICENSE file for applicable license.
   <target name="preprocess"
     dita:depends="{depend.preprocess.pre},
                   preprocess.init,
+                  preprocess-ditavals,
```

Comment:
IMO the `preprocess-` prefix is redundant and we don't use it in any other target name. (The init target doesn't count.)